topics/single-cell/tutorials/scatac-batch-correction-snapatac2/workflows/main_workflow.ga

```diff
@@ -0,0 +1,1675 @@
+{
```

Comment:
It would be great if you could add a workflow test: https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html

Reply:
thanks for the review! I'll add the test